templates/server/config/environment/test.js

```diff
@@ -3,7 +3,7 @@
 
   module.exports = {
     <% if(mongoose) { %>mongo: {
-      uri: 'mongodb://localhost/<%= appNameSlug %>-test'
+      uri: 'mongodb://localhost/<%= appNameSlug %>'
```

Comment:
I think this is helpful to have test run on a different DB. Can we change it back?